.github/workflows/test.yml

```diff
@@ -65,7 +54,7 @@ jobs:
           no_rate_limiter_exit=$?
           export TEST_RATE_LIMITER='true'
           pip install -r requirements-opts.txt
-          pytest -v --cov --cov-append
+          pytest -v --cov --cov-append -n auto --dist loadfile
```

Comment:
```suggestion
          # `-n auto --dist loadfile` uses pytest-xdist to run each test file on a different CPU worker
          pytest -v --cov --cov-append -n auto --dist loadfile
```

Reply:
can we add comments in that block?

Reply:
Yes, see [here](https://github.com/python-telegram-bot/python-telegram-bot/pull/3267/files#diff-faff1af3d8ff408964a57b2e475f69a6b7c7b71c9978cccc8f471798caac2c88)